test/core/reporters/test_realtime_reporter.pkb

```diff
@@ -112,7 +112,7 @@ create or replace package body test_realtime_reporter as
       -- consume
       select test_event_object(item_type, xmltype(text))
         bulk collect into g_events
-        from table(ut3.ut_output_table_buffer(l_reporter.output_buffer.output_id).get_lines())
+        from table(l_reporter.get_lines())
```

Comment:
@pesse
This is the type of call to retrieve reporter data I had in mind.
I had to change this test as it started failing when I changed reporter internal implementation (type of buffer used)